backend/README.md

```diff
@@ -26,10 +26,6 @@ Procedure:
     ```
     pipenv shell
     ```
-- Change to source code directory
```

Comment:
why this change is readme

Reply:
I havent made this chane must be some error